cloud-interfaces/api/cloudservers-api.rst

```diff
@@ -155,17 +155,16 @@ the request parameters named ``status``, ``image``, and ``flavor``
 correspond to the filters available on the Cloud Control Panel.
 
 In the
-:rax-docs:`Getting Started Guide for the Cloud Servers API <servers/api/v2/cs-gettingstarted>`,
-you can follow a 12-step tutorial to perform an essential task:
+:rax-docs:`Getting Started Guide for Cloud Servers - Create your first server <cloud-servers/v2/getting-started/create-server-intro/>`,
+you can follow a 3-step tutorial to perform an essential task:
 create a cloud server.
 The
-:rax-docs:`tutorial <servers/api/v2/cs-gettingstarted/content/ch_gs_getting_started_with_nova.html>`
+:rax-docs:`Getting Started Guide <cloud-servers/v2/getting-started/>`
 begins with instructions on creating a Rackspace account
-and concludes with deleting the Cloud Server that was created.
-An intermediate step
-demonstrates
-:rax-docs:`obtaining a detailed list of Cloud Servers by using the cURL command-line interface (CLI)
-<servers/api/v2/cs-gettingstarted/content/curl_list_servers.html>`.
+and walks you through basic server and network operations.
+An intermediate step shows an example with the cURL command-line interface (CLI) for
+:rax-docs:`Listing servers (cURL)
+<cloud-servers/v2/getting-started/create-server/listing-servers-curl/>`.
 
 Learn about Cloud Servers in SDK QuickStart
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
There are 2 links in this section to SDK QuickStart for Cloud Servers that gives a page not found..